sencha-workspace/SlateAdmin/app/controller/People.js

```diff
@@ -253,7 +255,7 @@ Ext.define('SlateAdmin.controller.People', {
                 }
 
                 me.selectPerson(person, function() {
-                    ExtHistory.resumeState();
+                    ExtHistory.resumeState(false);
```

Comment:
@themightychris I also noticed this behavior when browsing sections by course and department from settings in a branch where resumeState() is called without false.  I need to look into it further.